mindmeld/markup.py

```diff
@@ -105,37 +105,28 @@ def load_query_file(
         query_cache (QueryCache): A container containing cache query objects
 
     Returns:
-        ProcessedQuery: a processed query
+        List of cached query ids
     """
     query_factory = query_factory or QueryFactory.create_query_factory(app_path)
 
-    queries = []
+    query_ids = []
     for query_text in read_query_file(file_path):
         if query_text[0] == "-":
             continue
 
-        if query_cache:
-            query = query_cache.get_value(domain, intent, query_text)
-            if not query:
-                query = load_query(
-                    query_text,
-                    query_factory=query_factory,
-                    domain=domain,
-                    intent=intent,
-                    is_gold=is_gold,
-                )
-                query_cache.set_value(domain, intent, query_text, query)
-        else:
+        key = query_cache.get_key(domain, intent, query_text)
```

Comment:
It would be great if we can abstract out the details of the cache key gen and access through an API that preserves the previous call: `query_cache.get_value(domain, intent, query_text)`